osbuild/pipeline.py

```diff
@@ -329,7 +344,7 @@ def load(description, sources_options):
     if build:
         build_pipeline, runner = load_build(build, sources_options)
     else:
-        build_pipeline, runner = None, "org.osbuild.host"
+        build_pipeline, runner = None, "org.osbuild." + detect_os()
```

Comment:
Should we handle the case when this file doesn't exist? I'd result in more understandable error messages.

Reply:
I don't think the error is that hard to read, but I wouldn't be against adding a check. It should probably closer to where the runner is used though? Not sure yet, and I think independent of this change.